backend/iam/serializers.py

```
@@ -1,7 +1,14 @@
+from datetime import timedelta
+from django.utils import timezone
```

Ruff (0.8.2)

`django.utils.timezone` imported but unused

Remove unused import: `django.utils.timezone`

(F401)